backend/app/Repositories/UserRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\User;
+use App\Repositories\Contracts\UserRepository as IUserRepository;
+
+final class UserRepository implements IUserRepository
+{
+    public function create(array $fields): User
```

Comment:
name method save, and pass there User entity, that will be persisted and returned, that is save(User $user) { $user->save(); return $user; }